website/docs/schema-directives.mdx

```diff
@@ -121,7 +121,7 @@ To appreciate the range of possibilities enabled by `mapSchema`, let's examine a
 
 Suppose you want to ensure a string-valued field is converted to uppercase. Though this use case is simple, it's a good example of a directive implementation that works by wrapping a field's `resolve` function:
 
-```js
+```typescript
```

Comment:
Up to you if you want this to be in, but I thought it'd be best to format codeblocks with types as TS.